samples/subsys/usb/hid-mouse/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _usb_hid-mouse:
+
+USB HID mouse Sample Application
+####################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB HID driver provided by the zephyr
+project, enumerate a board into a mouse. But that mouse only has left button
+and can't move.
+This sample can be found under :file:`samples/subsys/usb/hid-mouse` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver, and there must has at least one
+gpio button in your board.
+
+Running
+********************
+
+Plug the board into a host device, for example, a PC running Linux.
```

Comment:
Change to:

    After you have built and flashed the sample app image to your board, plug
    the board into ...